### DISCUSSION TITLE
Smooth scrolling in macOS?

### DISCUSSION THREAD

**Discussion by clvnkhr** [ahrm/sioyek#331] [Q&A]
I was wondering if it is possible to make the mouse scrolling smooth/native, to match with the other apps I have open. Curiously, the scrolling in a Portal is already smooth. It would be enough for me if the scrolling was smooth only in the window that pops up when you right-click a reference, as the scrolling is too fast in this box with low height. 

Thank you for the wonderful viewer. I have put aside my work to learn how to use this properly. This may become a non-issue (personally) when I get more used to keyboard scrolling.

**Comment by ahrm** [comment]
You can adjust scroll speed by setting `vertical_move_amount` and `touchpad_sensitivity` options in `prefs_user.config`. The former sets all scroll speeds and the later is a multiplier when using touchpad or mouse wheel.

**Reply by clvnkhr** [reply]
I did see this in prefs but didn't try earlier. I have tried it now. While this is not the smooth scrolling, it has fixed all my issues. Thank you very much!
